packages/mdc-select/mdc-select.scss

```diff
@@ -177,3 +177,66 @@
     }
   }
 }
+
+/* postcss-bem-linter: define multi-select */
+
+/* stylelint-disable plugin/selector-bem-pattern */
+.mdc-multi-select {
+  appearance: none;
+  width: 250px;
+  padding: 0;
+  border: 1px solid rgba(black, .1);
+  border-bottom: 1px solid rgba(black, .1);
+  outline: none;
+
+  .mdc-list-group {
+    margin: 16px 0 0;
+    padding: 0 0 0 16px;
+    color: rgba(black, .38);
```

Comment:
Instead of hard-coding this color value you'll want to `@include mdc-theme-prop(color, text-hint-on-light)`.

You'll also want to account for this for a dark theme

```scss
.mdc-list-group {
  // ...
  @include mdc-theme-dark {
    @include mdc-theme-prop(color, text-hint-on-dark);
  }
}
```